corehq/apps/app_manager/id_strings.py

```diff
@@ -90,6 +90,16 @@ def detail_column_header_locale(module, detail_type, column):
     )
 
 
+@pattern('m%d.%s.%s_%s_%d.header')
+def invisible_column_header_locale(module, detail_type, column):
+    return u"m{module.id}.{detail_type}.{d.model}_{field}_{d_id}.header".format(
```

Comment:
This came from the need to have a different method call to identify its being done for invisible/sort-only fields. I was trying to update this to be like `m0.case_short.case_sort_only_field_name_1.display` to uniquely identify sort only string in app_strings but felt like over-do since it was just mapping.